schemas/json/theme.json

```diff
@@ -373,13 +324,18 @@
 				}
 			}
 		},
-		"settingsPropertiesPosition": {
+		"settingsPositionProperties": {
 			"type": "object",
 			"properties": {
 				"position": {
 					"description": "Settings related to position.",
 					"type": "object",
 					"properties": {
+						"fixed": {
+							"description": "Allow users to set fixed position.",
+							"type": "boolean",
+							"default": false
+						},
```

Comment:
Ah, thanks for checking the dev note. I just saw it in the global styles code next to `sticky`, so I assumed it was available. Checking the PR history, it doesn't like any blocks support `fixed` so that code, even though it exists, should never run. I'll remove it here, and assume that it'll be added when the block support is added.